src/Core/Universe.m

```diff
@@ -6305,6 +6305,7 @@ - (void) setScreenTextureDescriptorForKey:(NSString *)key descriptor:(NSDictiona
 	}
 	[screenBackgrounds release];
 	screenBackgrounds = [sbCopy copy];
+	[sbCopy release];
 }
```

Comment:
This one should be OK but it is not related to this PR. It should be submitted separately (and will needconfirmation that it still works as expected and without crashes).